tests/fixtures/testModel.ts

```diff
@@ -328,6 +328,7 @@ export class TestClassModel extends TestClassBaseModel {
   constructor(
     public publicConstructorVar: string,
     protected protectedConstructorVar: string,
+    defaultConstructorArgument: string,
```

Comment:
I'd like to prefix this with an underscore, but the linter yells at me if I do :disappointed: